test/files/rules/no-shadowed-variable.test.ts

```diff
@@ -75,3 +75,23 @@ function testDoStatement() {
         let foo = 2; // failure
     } while (true);
 }
+
+function testDestructuring() {
+    var x, y, z;
```

Comment:
Can you add some destructuring bindings to make sure they get added to the scope? e.g.

````typescript
var x;
var {y, z} = {y: 2, z: 3}
````